edx/edx-app-ios
# Ensure logical focus hierarchy for My Courses

### Description

[MA-2349](https://openedx.atlassian.net/browse/MA-2349)

Implemented `UIAlertController` for can't find courses button, and set voice over focus to navigation button when the alertController is closed.

### How to test this PR

1. Turn voice over on.
2. Double tap on "Can't find one of your courses?" button to open the UIAlertController.
3. Close to alertController to notice that the focus is properly set to top left navigation button.

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
